dispatcher_plugin_gw/dataserver_dispatcher.py

```diff
@@ -1,4 +1,5 @@
 import time
+from xml.etree.ElementTree import Comment
```

Comment:
what's this, auto-import?

Reply:
I think so. Clearly, not needed, thanks.